### ISSUE TITLE
Add indicators to canceled events

### ISSUE THREAD

**Issue by InciteDemocracy** [ResistanceCalendar/resistance-calendar-frontend#138]
Problem: FB Events that are canceled currently have no working link to the event page url, and give no indication they have been canceled. 

Solution: 

1. Add event page URLs for canceled events to API
2. Add (Canceled) to end of event title

**Comment by geoffreyyip** [comment]
How would the front-end application know if an event was cancelled on Fb? Are we planning to handle that in the API?

**Comment by InciteDemocracy** [comment]
Yes we need to update the API to include `is_canceled` I've created an issue for the back end here: https://github.com/ResistanceCalendar/resistance-calendar-api/issues/78I've created an issue

**Comment by aaghevli** [comment]
Hi folks. As per the following, please use the OSDI event.status field to determine cancelled status.

https://github.com/ResistanceCalendar/resistance-calendar-api/pull/89


```
status: { type: String, enum: ['confirmed', 'tentatives', 'cancelled'] }
```
